### DISCUSSION TITLE
score multiplier issue with standardised score

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#17349] [Q&A]
While checking some beatmaps leaderboards I see an issue with the ordering. I understand that the system is different and based on 30% acc 70% combo, but on this score its seem really unexpected and weird.

On website :
![image](https://user-images.githubusercontent.com/50368878/159127275-b8db60bd-c0e7-4cde-9844-5f2d43c322a6.png)

On lazer :
![image](https://user-images.githubusercontent.com/50368878/159127299-7e608a26-321e-49b3-9c8f-321c257220cb.png)

That a really big HD abuse here I don't know how its even possible.

map link :
https://osu.ppy.sh/beatmapsets/1299243#osu/2694749
(country leaderboard)

**Comment by bdach** [comment]
how is it standardised score if even the scores without mod multipliers are above 1 million

**Reply by Biokax** [reply]
because the classic display score mod is based on the standardised
![image](https://user-images.githubusercontent.com/50368878/159127935-d95d843f-410a-419f-a967-46e8bc9e500a.png)

**Reply by bdach** [reply]
well i don't have an answer for you. this is what flat multipliers inevitably lead to.

a known issue is that mod multipliers end up actually being squared in classic. but fixing this would cause classic scoring to end up with a different score ordering than standardised which was deemed not acceptable from a support/infrastructure perspective.

probably the best thing that could be done is a multiplier adjustment/rebalance.

**Reply by respektive** [reply]
I think it would make sense to use the mod multipliers scorev2 on stable uses. They have shown to work quite well with the 1mil score cap.

Though the HD multiplier is still the same but other mods like HR got adjusted to be higher than HD. In this case it wouldn't change anything. It has been mentioned before (i think in a community meeting) that the score ratios could be changed to be 50/50 for combo and acc instead of 70/30 in the future, which would probably prevent this kinda abuse in most cases.

**Reply by WitherFlower** [reply]
I think the problem OP is trying to point out here is that multipliers are too large for standardised scoring, so they change the ordering of leaderboards.

In stable they are applied to scorev1 (classic type), but in lazer they are applied to standardised scoring.

Maybe a solution would just be to halve/square root mod multipliers before applying them to standardised scoring, so ordering stays relatively constant.

**Comment by peppy** [comment]
This is one of the discussion points in the survey we are currently running, for what it's worth.

**Reply by Biokax** [reply]
alright, we will see with the survey. For this leaderboard, sort by pp solve the problem, however what about loved maps, do we have to make pp lb that will not count on the player profile ? Does the same thing will append with unranked maps ?
